webpack.config.shared.js

```diff
@@ -33,7 +33,6 @@ module.exports = {
     query: {
       "presets": [
         ["env", { "modules": false }],
-        "latest",
```

Comment:
What's the significance of this change?

Reply:
the `latest` preset is deprecated, see https://babeljs.io/docs/plugins/preset-latest/

not sure why we were using it at all, it has been deprecated for quite some time (I think since before we started this project iirc). the `env` preset takes its place, so using the `latest` preset here was pretty redundant.